Casks/a/atomic-wallet.rb

```diff
@@ -8,8 +8,8 @@
   homepage "https://atomicwallet.io/"
 
   livecheck do
-    url "https://releases.atomicwallet.io/latest-mac.yml"
-    strategy :electron_builder
+    url "https://releases.atomicwallet.io/download/latest-mac.txt"
+    regex(/(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/v?(\d+(?:[.-]\d+)+)/i)
```

Older releases used a version scheme with a suffix like `1.2.3-456`, so we should account for that here. Due to the looseness of this regex, it would otherwise match the `1.2.3` part and omit the suffix.